simpeg/electromagnetics/static/utils/static_utils.py

```diff
@@ -1521,15 +1524,23 @@ def r_unit(p1, p2):
     return vec, r
 
 
-def gettopoCC(mesh, ind_active, option="top"):
+@deprecated(
+    "The `gettopoCC` function is deprecated, "
+    "and will be removed in SimPEG v0.26.0. "
```

Comment:
We remove bits of code after two minor releases since the deprecation warning as minimum, so we better remove this in v0.27.0.
```suggestion
    "and will be removed in SimPEG v0.27.0. "
```